arcgis-ios-sdk-samples/Content Display Logic/Model/SearchEngine.swift

```diff
@@ -81,7 +81,7 @@ class SearchEngine: NSObject {
     }
     
     
-    private func indexAllReadmes(readmeDirectoriesURLs:[NSURL]) {
+    fileprivate func indexAllReadmes(_ readmeDirectoriesURLs:[URL]) {
```

Comment:
Removed the `_`. Usage now: `weakSelf.indexAllReadmes(readmeDirectoriesURLs: readmeDirectoriesURLs)`